modules/ROOT/pages/grub-password.adoc

```diff
@@ -0,0 +1,35 @@
+= Setting a GRUB password
+
+Some FCOS users may want to setup a password to prevent unauthorized users from accessing the GRUB command line, modifying kernel command-line arguments, or booting non-default OSTree deployments. The steps below will outline the process of setting a GRUB password.
```

Comment:
```suggestion
You can setup a GRUB password to prevent unauthorized users from accessing the GRUB command line, modifying kernel command-line arguments, or booting non-default OSTree deployments. The process is outlined in the steps below:
```

We may even want to remove the ` The process is outlined in the steps below:` part.